src/app/screens/confirmBrc20Transaction/confirmBrc20TransactionComponent.tsx

```diff
@@ -0,0 +1,236 @@
+import ActionButton from '@components/button';
+import BigNumber from 'bignumber.js';
+import Brc20FeesComponent from '@screens/confirmBrc20Transaction/brc20FeesComponent';
+import RecipientComponent from '@components/recipientComponent';
+import SettingIcon from '@assets/img/dashboard/faders_horizontal.svg';
+import TransactionDetailComponent from '@components/transactionDetailComponent';
+import styled from 'styled-components';
+import useWalletSelector from '@hooks/useWalletSelector';
+import { NumericFormat } from 'react-number-format';
+import { Recipient } from '@secretkeylabs/xverse-core/transactions/btc';
+import { FungibleToken, getBtcFiatEquivalent } from '@secretkeylabs/xverse-core';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { EditFees, OnChangeFeeRate } from './editFees';
```

Comment:
```suggestion
import type { OnChangeFeeRate } from './editFees';
import { EditFees } from './editFees';
```

Reply:
is there a functional difference to splitting the type out to import type like this, when you're already importing a function as well?